controllers/storagecluster/storageclasses.go

```diff
@@ -280,6 +281,46 @@ func newCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) S
 	}
 }
 
+// newVirtualizationCephBlockPoolStorageClassConfiguration generates configuration options for a Ceph Block Pool StorageClass for virtualization.
+
+func newVirtualizationCephBlockPoolStorageClassConfiguration(initData *ocsv1.StorageCluster) StorageClassConfiguration {
+	persistentVolumeReclaimDelete := corev1.PersistentVolumeReclaimDelete
+	allowVolumeExpansion := true
+	managementSpec := initData.Spec.ManagedResources.CephBlockPools
+	return StorageClassConfiguration{
+		storageClass: &storagev1.StorageClass{
+			ObjectMeta: metav1.ObjectMeta{
+				Name: generateNameForVirtualCephBlockPoolSC(initData),
```

Comment:
"Virtual" is a very overloaded term.  "Virt" would be a better shorthand here IMO (or just spell it out entirely).